pvlib/pvsystem.py

```diff
@@ -534,11 +534,11 @@ def retrieve_sam(name=None, samfile=None):
         name = name.lower()
 
         if name == 'cecmod':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-cec-modules-2014-1-14.csv'
+            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-cec-modules-2015-6-30.csv'
         elif name == 'sandiamod':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-modules-2014-1-14.csv'
-        elif name == 'sandiainverter':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-inverters-2014-1-14.csv'
+            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-modules-2015-6-30.csv'
+        elif name == 'sandiainverter': # Still only one current inverter set, so left it as sandia, to avoid breaking old code
```

Comment:
Personally I don't see much value in old versions, because if you are paying that close attention to the different module or inverter parameters, you would probably do better to have a local copy that you version yourself, and can specify directly as a file (and would have to update the column names to match, but that's not hard at this point).